camera_system/block_to_intersection.py

```diff
@@ -72,9 +82,11 @@ def get_cost(self) -> float:
 
         # 動作時間に回頭時間を足す（成功率に変動はなし）
         m_time += self.__rotation_time
-        # 回頭している場合，回頭前後のスリープ時間を足す
-        if self.__rotation_angle != 0:
-            m_time += GameMotion.SLEEP_TIME * 2
+        # 回頭前後のスリープ時間を足す
+        m_time += GameMotion.SLEEP_TIME * 2
+        # 方向転換する、かつ、角度補正する場合、スリープが1回増える
+        if self.__rotation_angle != 0 and self.__can_correction:
+            m_time += GameMotion.SLEEP_TIME
```

Comment:
回頭も補正もない場合はスリープ時間は足されないんじゃない?
あと同じ条件ってことが分かりやすいように56行目とかと書き方揃えた方がいいかも
```suggestion
        if self.__can_correction or self.__rotation_angle != 0:
            m_time += GameMotion.SLEEP_TIME * 2
        # 方向転換する、かつ、角度補正する場合、スリープが1回増える
        if self.__can_correction and self.__rotation_angle != 0:
            m_time += GameMotion.SLEEP_TIME
```